packages/create/src/cli/add/component-usages.ts

```diff
@@ -0,0 +1,52 @@
+import type { Command } from 'commander';
+import { generateChange, getVariant, ChangeType, getPromptsForAddComponentUsages } from '@sap-ux/adp-tooling';
+import { getLogger, traceChanges } from '../../tracing';
+import { validateAdpProject } from '../../validation/validation';
+import { promptYUIQuestions } from '../../common';
+
+/**
+ * Add a new sub-command to change the data source of an adaptation project to the given command.
+ *
+ * @param {Command} cmd - The command to add the change data-source sub-command to.
+ */
```

Comment:
This is the command for `add component usages`.

Reply:
Fixed with: 4ea2aa9c0137314014ddeeee2e8e1df80149357b